src/hyperbolic_system.hpp

```diff
@@ -208,12 +297,7 @@ template <FluxDir DIR>
 void HyperbolicSystem<problem_t>::ReconstructStatesPPM(amrex::MultiFab const &q_mf, amrex::MultiFab &leftState_mf, amrex::MultiFab &rightState_mf,
 						       const int nghost, const int nvars, const int iReadFrom, const int iWriteFrom)
 {
-	BL_PROFILE("HyperbolicSystem::ReconstructStatesPPM()");
-
-	// By convention, the interfaces are defined on the left edge of each
-	// zone, i.e. xleft_(i) is the "left"-side of the interface at the left
-	// edge of zone i, and xright_(i) is the "right"-side of the interface
-	// at the *left* edge of zone i.
+	BL_PROFILE("HyperbolicSystem::ReconstructStatesPPM(MultiFabs)");
```

Comment:
You should be able to fix the clang-tidy warning by changing this to
`const BL_PROFILE("HyperbolicSystem::ReconstructStatesPPM(MultiFabs)");`

Reply:
Done.